docs/toucan-form/changeset-validation/demo/base-demo.md

```diff
@@ -4,7 +4,7 @@
   @data={{changeset this.data this.validations}}
   @dataMode='mutable'
   @onSubmit={{this.handleSubmit}}
-  @validate={{validate-changeset}}
+  @validate={{(validate-changeset)}}
```

Comment:
Just for my own knowledge, what's the `()` add? Calling the `validate-changeset`?

Reply:
Exactly!

It's a bit weird tbh. For some earlier versions of ember this syntax didn't exist IIRC. It was added so you can call a helper without any arguments. Compare this to e.g. `{{validate-changeset foo=bar}}`. This is also calling, because there is no way to pass arguments to a function/helper other than calling it. And `{{validate-changeset}}` is used to pass things as a value, so in this case passing the helper function, instead of calling it. So Ember needed an explicit way to call a helper, when no args are expected, and this is the syntax for that!